disco/types/message.py

```diff
@@ -385,40 +406,47 @@ class Message(SlottedModel):
         Whether this is a TTS (text-to-speech) message.
     mention_everyone : bool
         Whether this message has an @everyone which mentions everyone.
-    pinned : bool
-        Whether this message is pinned in the channel.
     mentions : dict[snowflake, `User`]
         Users mentioned within this message.
     mention_roles : list[snowflake]
         IDs for roles mentioned within this message.
-    embeds : list[`MessageEmbed`]
-        Embeds for this message.
     attachments : dict[`MessageAttachment`]
         Attachments for this message.
+    embeds : list[`MessageEmbed`]
+        Embeds for this message.
     reactions : list[`MessageReaction`]
         Reactions for this message.
+    nonce : str
+        The nonce of this message.
+    pinned : bool
+        Whether this message is pinned in the channel.
+    webhook_id : snowflake
+        The id if the message is a webhook.
+    type : `MessageType`
+        Type of the message.
     activity : `MessageActivity`
         The activity of a Rich Presence-related chat embed.
     application : `MessageApplication`
         The application of a Rich Presence-related chat embed.
     """
     id = Field(snowflake)
     channel_id = Field(snowflake)
-    webhook_id = Field(snowflake)
-    type = Field(enum(MessageType))
+    guild_id = Field(snowflake)
```

Comment:
Adding guild_id here seems to be leaving event.guild_id as an unset for the MessageCreate and MessageUpdate events.

Reply:
If that's where that bug came from, you're a saint, otherwise the bug's existed for some time now, however, I hope removing it works. Will make the adjustment when I'm at the PC.